code/src/Pages/Shopping.js

```diff
@@ -0,0 +1,15 @@
+
+import React from 'react'
+import { HeaderShopping } from '../components/HeaderShopping'
+import { ShoppingList } from '../components/ShoppingList'
+
+
+export const Shopping = () => {
+
+  return (
+    <>
+      <HeaderShopping />
+      <ShoppingList />
+    </>
+  )
```

Comment:
this could be written as an implicit return, since you're only return something, not doing any other logics in here
```
export const Shopping = () => (
    <>
      <HeaderShopping />
      <ShoppingList />
    </>
  )
```
A little [reading](https://riptutorial.com/javascript/example/17665/implicit-return) about this